src/main/java/racingcar/RacingGameMain.java

```diff
@@ -0,0 +1,44 @@
+package racingcar;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.Scanner;
+import static racingcar.CarConstant.*;
+
+public class RacingGameMain {
+
+    public static void main(String[] args) {
+
+        Scanner scanner = new Scanner(System.in);
+        RacingGame racingGame = new RacingGame();
+        WinnerExtraction winnerExtraction = new WinnerExtraction();
+
+        System.out.println("경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분).");
+        String userInput = scanner.nextLine();
+        String[] carNameArr = userInput.split(",");
```

Comment:
```suggestion
        String[] carNameArr = userInput.trim().split(",");
```
저번 피드백에 잘 못 설명드렸습니다. 😭